src/Logger/ElasticaLogger.php

```diff
@@ -90,6 +91,7 @@ public function logQuery($path, $method, $data, $queryTime, $connection = [], $q
                 'connection' => $connection,
                 'queryString' => $query,
                 'itemCount' => $itemCount,
+                'itemCountIsAccurate' => $itemCountIsAccurate,
```

Comment:
What about `itemCountRelation` instead?
We can just display the result from elastic directly instead of converting to a bool.

Reply:
Yes it could be an other option, I chose to use a boolean because the `relation` returned by elastic can have only two values `eq` or `gte`, but I can update this PR if you prefer this parameter to be a string ?